docs/issue-management.md

```diff
@@ -0,0 +1,395 @@
+  * discuss any impacts to existing APIs
+
+#### Design
+
+A `kind/design` label categorizes issue or PR as related to design.
+
+A design issue or PR is for discussing larger architectural and design proposals.
+Approval of a design proposal may result in multiple additional feature,
+api-change, or cleanup issues being created to implement the design.
+
+To create a design issue:
+
+* label your issue or PR with `kind/design`
+* describe the design in the issue or PR body
+
+Before creating additional issues or PRs that implement the proposed design it is
+important to get feedback and approval from the maintainers. Design feedback
+could include some of the following:
+
+* needs additional detail
+* no, this problem should be solved in another way
+* this is desireable but we need help completing other issues or PRs first; then we will
+  consider this design
+
+_Note this was previously called `proposal`. We will add the `kind/design` label
+to all issues currently labeled with `proposal`._
+
+#### Documentation
+
+A `kind/documentation` label categorizes issue or PR as related to a
+documentation.
+
+To create a documentation issue or PR:
+
+* label your issue or PR with `kind/documentation`
+* title the issue with a short description of what you are documenting
+* provide a brief summary in the issue or PR body of what you are documenting. In some
+  cases, it might be useful to include a checklist of changed documentation
+  files to indicate your progress.
+
+#### Failing Test
+
+A `kind/failing-test` label categorizes issue or PR as related to a consistently
+or frequently failing test.
+
+To create a failing test issue or PR:
+
+* label your issue or PR with `kind/failing-test`
+
+TODO: As more automation is used in the continuous integration pipeline, we will
+be able to automatically generate an issue for failing tests.
+
+#### Support
+
+A `kind/support` label categorizes issue as related to a support request.
+
+To create a support issue or PR:
+
+* label your issue or PR with `kind/support`
+* title the issue or PR with a short description of your support request
+* answer all of the questions in the support issue template
+
+_Note this was previously called `support`. We will add the `kind/support` label
+to all issues currently labeled with `support`._
+
+### Area
+
+Area labels begin with `area/<area>` and identify areas of interest or functionality
+to which an issue relates. An issue or PR could have multiple areas. These labels are
+used to sort issues and PRs into categories such as:
+
+* operating systems
+* cloud platform, 
+* functional area,
+* operating or legal area (i.e., licensing), 
+* etc.
+
+A list of areas is maintained in [`docs/github-labels.md`](docs/github-labels.md).
+
+An area may be changed, added or deleted during issue or PR triage.
+
+### Size
+
+Size labels beting with `size/<size>` and estimate the relative complexity or work
```

Comment:
s/beting/begin

Reply:
Corrected. Not sure why spell check plugin not picking this up. Will investigate.

Reply:
It's my understanding that the spell checker only catches the most common spelling mistakes and does not lookup each work in a dictionary, as I would image there would be too many false positives for technical documentation.